docs/site/content/docs/assets/install-package-to-cluster.md

```diff
@@ -0,0 +1,75 @@
+1. List the available package repositories.
+
+    ```sh
+    tanzu package repository list --all-namespaces
+    ```
+
+1. Verify the `STATUS` is `Reconcile succeeded` in the output of the above
+   command.
+
+    ```txt
+    NAME                 REPOSITORY                                           TAG                     STATUS               DETAILS  NAMESPACE
+    tkg-core-repository  projects.registry.vmware.com/tce/main                0.9.1                   Reconcile succeeded           tanzu-package-repo-global
```

Comment:
```suggestion
    tkg-core-repository  projects.registry.vmware.com/tce/main                {{< pkg_repo_latest >}}                   Reconcile succeeded           tanzu-package-repo-global
```

Reply:
@kcoriordan, unfortunately the newest package repo is not necessarily what unmanaged is going to ship with, in fact, it'll ship out of the gate with 0.10.0.